src/app/MainPage.tsx

```diff
@@ -4,7 +4,7 @@ import { useAtom } from "jotai";
 import { messageAtom } from "@/store/messageAtom.tsx";
 
 function MainPage() {
-  const [message, setMessage] = useAtom(messageAtom);
+  const [message] = useAtom(messageAtom);
```

Comment:
변수를 없애는 것도 방법인데, 추후에 사용하는거면 임시로 이런 방법도 있어요!

```suggestion
  const [message, _] = useAtom(messageAtom);